.gitlab-ci.yml

```diff
@@ -29,13 +24,15 @@ criterion-benchmark:
   - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                   # PRs
   <<:                              *docker-env
   script:
+    - cargo install cargo-criterion
     - git fetch
     - git submodule update --init --recursive
     - git checkout master
     - git pull
-    # on master
-    - cargo bench --bench benches -- --noplot --save-baseline master | tee bench-report-master.txt
-    # on PR
+    - cd wasmi_v1
+    # benchmark master
+    - cargo criterion --message-format=json | tee -a benchmarks.json
+    # benchmark PR
     - git checkout $CI_COMMIT_SHA
-    - cargo bench --bench benches -- --noplot --baseline master | tee bench-report-pr.txt
-    - bash ./scripts/ci/benchmarks-report.sh bench-report-master.txt bench-report-pr.txt
+    - cargo criterion --message-format=json | tee -a benchmarks.json
+    - bash ../scripts/ci/benchmarks-report.sh benchmarks.json
```

Comment:
I wonder if it was useful to have a dedicated folder for temporary files such as the `benchmarks.json`. Just imagine the case when we have a `benchmarks.json` in the Git root folder. Then this would break. Same goes with any temporary file that might be shadowed. Since Rust already uses the `target` folder we could put those CI related files under `target/ci/benchmark.json`? What do you think?

Reply:
Actually this is not so important. Because after pipeline finishes those files are gone. Next pipeline execution will start from scratch.
If there is a need to keep those files for further investigation, we can save them as pipeline artifacts in GitLab.

Reply:
I know. I still think it would be good to have a place where we put temporary files instead of just flooding them into the root folder. Flooding in root folder bears the risk over overwriting actual non-temporary files.

Reply:
Now temp files will reside in `./target/ci` folder

Reply:
Cool, resolved then! :+1: